ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,424 @@
+ansible-playbook playbooks/deploy.yml
+```
+
+### EOS Intended Configurations
+
+Your configuration files should be similar to these.
+
+=== "SPINE1"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/SPINE1.cfg
+    --8<--
+    ```
+
+=== "SPINE2"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/SPINE2.cfg
+    --8<--
+    ```
+
+=== "LEAF1"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/LEAF1.cfg
+    --8<--
+    ```
+
+=== "LEAF2"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/LEAF2.cfg
+    --8<--
+    ```
+
+=== "LEAF3"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/LEAF3.cfg
+    --8<--
+    ```
+
+=== "LEAF4"
+
+    ```shell
+    --8<--
+    examples/l2ls-fabric/intended/configs/LEAF4.cfg
+    --8<--
+    ```
+
+## Add Routing to Spines
+
+In our example, we used an external L3/FW Device to route between subnets. This is very typical in a Layer 2 only environment. To route on the spines, we remove the L3/FW device from the topology and create the SVIs on the Spines. The updated topology is shown below.
+
+???+ note
+
+    The spine type has been changed to **l3spine**.
+
+![Figure: 3](images/L2LS_Spine_routing.svg)
+
+The following group_vars need updating to enable L3 routing on the Spines.
+
+- DC1_SPINES.yml
+- DC1_FABRIC.yml
+- DC1_NETWORK_SERVICES.yml
+
+The updated changes are noted in the tabs below.
+
+=== "DC1_SPINES.yml"
+
+    Update type to `l3spine`. This makes it a routing device.
+
+    ``` yaml
+    ---
+    ### group_vars/DC1_SPINES.yml
+
+    type: l3spine
+    ```
+
+=== "DC1_FABRIC.yml"
+
+    Update with the following changes and additions.
+
+    - Change the node key **spine** to **l3spine** to match the node type set previously in DC1_SPINES.yml
+    - Add **loopback_ipv4_pool**
+    - Add **mlag_peer_l3_ipv4_pool**
+    - Add **virtual_router_mac_address**
+
+    Update DC1_FABRIC.yml with the following recommended settings. Use your own IP pools.
+
+    ``` yaml
+    # Node Key must be l3spine
+    l3spine:
+      defaults:
+        platform: cEOS-LAB
+        spanning_tree_mode: mstp
+        spanning_tree_priority: 4096
+        # Loopback is used to generate a router-id
+        loopback_ipv4_pool: 1.1.1.0/24
+        mlag_peer_ipv4_pool: 192.168.0.0/24
+        # Needed for L3 peering across the MLAG Trunk
+        mlag_peer_l3_ipv4_pool: 10.1.1.0/24
+        # Used for SVI Virual MAC address
+        virtual_router_mac_address: 00:1c:73:00:dc:01
+        mlag_interfaces: [Ethernet47, Ethernet48]
+    ```
+
+=== "DC1_NETWORK_SERVICES.yml"
+
+    Update Network Services to use L3 SVIs.
+
+    ???+ Note
+
+        To create L3 SVIs on the spines, we need to utilize an L3 VRF. In our case, we will use the default VRF. `VLANS` is simply a name for organizing VRFs and SVIs. You may change it to your liking.
+
+    ``` yaml
+    ---
+    tenants:
+      VLANS:
```

Comment:
I would strongly suggest a different name. Call it `EXAMPLE` or `MYFABRIC` or something to avoid confusing the user by bringing vlans into this.

Reply:
What bothers me about this is the tenants key name.  Isn't there a place where we can change the name of `tenants:`?

Reply:
Tenants does not make sense for an L2LS topo like it does for EVPN Fabric.

Reply:
Yes you can, but it will not be simple for a user. We have `network_services_keys`

Reply:
Is this possible to change tenants key with?

```
network_services_keys:
  - name: < key_1 >
  - name: < key_2 >
```

Reply:
changed to MY_FABRIC.